### DISCUSSION TITLE
Perspective workspace documentation

### DISCUSSION THREAD

**Discussion by aszenz** [perspective-dev/perspective#2555] [Q&A] [answered]
I accidentally stumbled onto perspective workspace and found it really useful, but I can't find the docs for it, am i missing them or simply non-existent yet.
For now reading the code was good enough for my work

**Comment by timkpaine** [comment, marked as answer]
This is deliberate as perspective-workspace is still under active development and it's APIs are not yet stable.

**Reply by aszenz** [reply]
Makes sense then, not sure if you are looking for any feedback already but I would like the ability to join tables and create new views, that would make the workspace feature complete with other paid dashboard builders

**Reply by texodus** [reply]
@aszenz I'm not interested in Perspective competing with paid dashboard builders. If you're interested in dashboard builders, come see what we're building on the Enterprise side at [Prospective](https://prospective.co/).

**Reply by aszenz** [reply]
Nice, I see great potential in perspective, having a company focus on it full time is promising.
